tests/e2e/files/rulebooks/test_disabled_rules.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Ruleset 1
+  hosts: all
+  sources:
+    - generic:
+        event_delay: 0.5
+        payload:
+          - action: "trigger_enabled_rule"
+          - action: "trigger_disabled_rule"
+
+  rules:
+    - name: Enabled rule
+      enabled: true
```

Comment:
I wonder if it supports variable substitution.

Reply:
Good question. The schema doesn't allow it. @mkanoor, what do you think about allowing variable substitution or the `enabled` param?

Reply:
@ttuffin I am not sure about it, so far only the name has substitution enabled. We can discuss in the dev sync call

Reply:
I think it could be good. Let's discuss it there.